serde_arrow/src/internal/serialization_ng/bool_builder.rs

```diff
@@ -39,7 +39,7 @@ impl SimpleSerializer for BoolBuilder {
         Ok(())
     }
 
-    fn serialize_none(&mut self) -> Result<()> {
+    fn serialize_unit(&mut self) -> Result<()> {
```

Comment:
TBH: I feel that `serialize_none` is semantically more correct than `serialize_unit` for missing values. Therefore, I would prefer to keep  `serialize_none`  as the impl in the builders, but use the default of `serialize_unit` to forward to `serialize_none`. (That would also result in less changes :))

Reply:
Done in local branch